src/IssueViz.Security.UnitTests/OpenInIDE/Http/ShowHotspotOwinRequestHandlerTests.cs

```diff
@@ -34,6 +34,8 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.OpenInIDE
     [TestClass]
     public class ShowHotspotOwinRequestHandlerTests
     {
+        const string ORIGIN = "http://origin";
```

Comment:
Just FYI, c# naming convention for constants is PascalCase :) we have a rule for that in the QP but it's test code so unfortunately it doesn't get picked up... I'll fix it for you 🙂